core/rpc/json/responses.go

```diff
@@ -8,11 +8,20 @@ import (
 // This file defines the response types. There is one for each request type.
 // As with the request types, many are based on types in the core module.
 
+type VersionResponse struct {
+	Service     string `json:"service"`
+	Version     string `json:"api_ver"`
+	Major       uint32 `json:"major"`
+	Minor       uint32 `json:"minor"`
+	Patch       uint32 `json:"patch"`
+	KwilVersion string `json:"kwil_ver"`
+}
+
 // AccountResponse contains the response object for MethodAccount.
 type AccountResponse struct {
-	Identifier []byte `json:"identifier,omitempty"`
-	Balance    string `json:"balance"`
-	Nonce      int64  `json:"nonce"`
+	Identifier types.HexBytes `json:"identifier,omitempty"`
```

Comment:
I just noticed that calling user.account only returns `{balance, nonce}`, now, and no longer includes the identifier. Is this intentional?

E.g. calling with postman:
`data: { jsonrpc: '2.0', id: 1, result: { balance: '0', nonce: 0 } }`

Reply:
For an account that does not exist, yes, that was intentional.  Can change though.  I also though it was pointless to return identifier that you provided in the request?

Reply:
Raises other questions.  Discussing in https://kwilteam.slack.com/archives/C05RZNFKNJD/p1715285006767559?thread_ts=1715284990.944059&cid=C05RZNFKNJD

Regardless of this response, I think now is a good time to change anything for the better since we're breaking quite a bit.  Only reason to minimize changes is to keep kwil-cli with `--output json` showing similar responses, but that's fairly niche I think.